apps/mobile/src/components/TradeChecklistFlow/components/CalculateAmountFlow/components/CalculateAmountScreen/index.tsx

```diff
@@ -86,16 +88,19 @@ function CalculateAmountScreen({
         void submitTradeChecklistUpdates()().finally(() => {
           showLoadingOverlay(false)
         })
+        navigation.navigate('ChatDetail', store.get(chatWithMessagesKeys))
+      } else {
+        goBack()
```

Comment:
Is this correct? Can `navigation.navigate('AgreeOnTradeDetails')` be called instead?